test/test_dataloader.py

```diff
@@ -1892,7 +1894,10 @@ def test_proper_exit(self):
             #   - `None` means that no error happens.
             # In all cases, all processes should end properly.
             if use_workers:
-                exit_methods = [None, 'loader_error', 'loader_kill', 'worker_error', 'worker_kill']
+                # TODO: Fix test for 'loader_kill' that would cause running out of shared memory.
+                # Killing loader process would prevent DataLoader iterator clean up all queues
+                # and worker processes
+                exit_methods = [None, 'loader_error', 'worker_error', 'worker_kill']
                 persistent_workers = self.persistent_workers
             else:
                 exit_methods = [None, 'loader_error', 'loader_kill']
```

Comment:
QQ: Just to confirm that `loader_kill` only has issue when `use_workers==True`?

Reply:
Yeah. It only has issue with multiprocessing -> shared memory used and not cleaned up properly.
When `use_workers=False`, `DataLoader` would only use single process. So, it should be fine.